tools/ARIAtools/product.py

```diff
@@ -12,11 +12,15 @@
 import logging
 import datetime
 import itertools
+import h5py
 import netCDF4
 import numpy as np
 import osgeo
+from pyproj import Transformer
 
 import shapely.geometry
+from shapely.ops import transform
+from shapely.wkt import loads
```

Comment:
Please use the full namespace and avoid the from X import Y pattern. A reader of the code will not know where these objects are coming from (Transformer, transform, loads)

Reply:
@alexfore I tried this, but get this error:
```
>>> import shapely.ops.transform
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'shapely.ops.transform'; 'shapely.ops' is not a package
```

How about this instead so users can get an idea of the provenance as you advise:
from shapely.ops import transform as shapely_ops_transform

Reply:
I'm saying do this
```
import shapely.geometry
import shapely.ops
import shapely.wkt
```
and use the full namespace in the function calls (`transform` and `loads`), also may not need `import shapely.geometry` as `import shapely` seems to import `shapely.geometry` already

Reply:
Ok done. I specified as `import shapely.geometry` instead of `import shapely`